ws-nextjs-app/pages/[service]/send/[id]/FormField/index.tsx

```diff
@@ -0,0 +1,53 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { PropsWithChildren, ReactElement } from 'react';
+import { FormComponentProps, InputProps } from '../types';
+import styles from '../styles';
+
+const Label = ({ id, children }: PropsWithChildren<{ id: string }>) => (
+  <label htmlFor={id}>{children}</label>
+);
+
+const TextInput = ({ id, name }: InputProps) => (
+  <input id={id} name={name} type="text" />
+);
+
+const TextArea = ({ id, name }: InputProps) => <textarea id={id} name={name} />;
+
+const EmailInput = ({ id, name }: InputProps) => (
+  <input id={id} name={name} type="email" />
+);
+
+const Checkbox = ({ id, name }: InputProps) => (
+  <input id={id} name={name} type="checkbox" />
+);
+
+const Telephone = ({ id, name }: InputProps) => (
+  <input id={id} name={name} type="tel" />
+);
+
+const FormComponents: Record<
+  string,
+  ({ id, name }: InputProps) => ReactElement
+> = {
+  text: TextInput,
+  email: EmailInput,
+  checkbox: Checkbox,
+  phone: Telephone,
+  textarea: TextArea,
```

Comment:
Just thinking we'll need to have a input of type `file` to allow for file uploads.